views/error/purged.tt

```diff
@@ -0,0 +1,77 @@
+[%# Page to print when trying to view information about a purged user
+  #
+  # Authors:
+  #     Jen Griffin <kareila@livejournal.com>
+  #
+  # Copyright (c) 2020 by Dreamwidth Studios, LLC.
+  #
+  # This program is free software; you may redistribute it and/or modify it under
+  # the same terms as Perl itself.  For a copy of the license, please reference
+  # 'perldoc perlartistic' or 'perldoc perlgpl'.
+%]
+
+[%- CALL dw.request_status( 410 ); # 410 Gone -%]
+
+[%- sections.title = 'error.purged.name' | ml -%]
+[%- sections.windowtitle = 'error.purged.title' | ml -%]
+[%- CALL dw.active_resource_group( "foundation" ) -%]
+
+<p>[% 'error.purged.text' | ml %]</p>
+
+<!--
+xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
```

Comment:
Wow, what?!

I dug -- it looks like this dates back to pauamma's original 404 BML (b1fa950341af0efbabaefcd8c899a674528a44fe). Reckon they'd remember why? I have no idea what this was meant to accomplish 11 years ago, but it has a strong smell of some BML-specific limitation. 

Oh no, wait, I dug further. Apache/LiveJournal.pm: 

```perl
    # Set to 1 if the code should generate junk to help IE
    # display a more meaningful error message.
    my $generate_iejunk = 0;

# ...

    $html .= ( "<!-- xxxxxxxxxxxxxxxxxxxxxxxxxxxx -->\n" x 100 ) if $generate_iejunk;
```

https://github.com/apparentlymart/livejournal/commit/3cb9c74f3f469ab069f82fb8d4edf13ced00d1b5

2003. It's something to do with IE 5 or 6.

I strongly suspect we don't want to continue doing this.

Reply:
Unfortunately, my research indicates that not only could this still be necessary, but that Chrome may have also flirted with joining the party of ignoring the content of error pages if the source is less than 512 bytes.

https://stackoverflow.com/questions/53499913/ie-turn-off-friendly-http-error-messages-via-server
https://stackoverflow.com/questions/11544048/how-do-i-suppress-friendly-error-messages-in-internet-explorer
https://superuser.com/questions/202244/show-http-error-details-in-google-chrome
https://bugs.chromium.org/p/chromium/issues/detail?id=1695

Reply:
But then, I just realized that these pages are well over 512 bytes because they're being rendered in site scheme. Not sure if that's the case for every error page on the site, but for the ones going through DW::Template, we can definitely pull it.